app/routes/project/detail/index.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  beforeModel() {
```

Comment:
@locks Would `redirect` be the preferred hook in this instance because there is no `model` hook?